packages/terra-overlay/src/Overlay.scss

```diff
@@ -0,0 +1,30 @@
+@import '~terra-mixins';
+
+.terra-Overlay {
+  background-color: #1c1f21;
```

Comment:
You can use `$terra-dark` here. It has a [default value](https://github.com/cerner/terra-core/blob/master/packages/terra-legacy-theme/src/_palette.scss#L90) of `#1c1f21`background-color: $terra-dark;`